xbmc/guilib/StereoscopicsManager.cpp

```diff
@@ -546,44 +549,11 @@ void CStereoscopicsManager::OnPlaybackStarted(void)
   {
   case STEREOSCOPIC_PLAYBACK_MODE_ASK: // Ask
     {
-      CApplicationMessenger::Get().MediaPause();
-
-      CGUIDialogSelect* pDlgSelect = (CGUIDialogSelect*)g_windowManager.GetWindow(WINDOW_DIALOG_SELECT);
-      pDlgSelect->Reset();
-      pDlgSelect->SetHeading(g_localizeStrings.Get(36527).c_str());
-
-      int idx_playing   = -1;
-
-      // add choices
-      int idx_preferred = pDlgSelect->Add(g_localizeStrings.Get(36524) // preferred
-                                     + " ("
-                                     + GetLabelForStereoMode(preferred)
-                                     + ")");
-
-      int idx_mono = pDlgSelect->Add(GetLabelForStereoMode(RENDER_STEREO_MODE_MONO)); // mono / 2d
-
-      if (playing != RENDER_STEREO_MODE_OFF && playing != preferred && preferred != RENDER_STEREO_MODE_AUTO && g_Windowing.SupportsStereo(playing)) // same as movie
-        idx_playing = pDlgSelect->Add(g_localizeStrings.Get(36532)
-                                    + " ("
-                                    + GetLabelForStereoMode(playing)
-                                    + ")");
-
-      int idx_select = pDlgSelect->Add( g_localizeStrings.Get(36531) ); // other / select
-
-      pDlgSelect->DoModal();
-
-      if(pDlgSelect->IsConfirmed())
-      {
-        int iItem = pDlgSelect->GetSelectedLabel();
-        if      (iItem == idx_preferred) mode = preferred;
-        else if (iItem == idx_mono)      mode = RENDER_STEREO_MODE_MONO;
-        else if (iItem == idx_playing)   mode = RENDER_STEREO_MODE_AUTO;
-        else if (iItem == idx_select)    mode = GetStereoModeByUserChoice();
-
-        SetStereoModeByUser( mode );
-      }
-
-      CApplicationMessenger::Get().MediaUnPause();
```

Comment:
I'm not sure if this really fix the issue because if the fullscreen window message needs a bit more time as this one we have the same result. We should think about a way to force activate a window or do not check for modals in case we activate fullscreen/visualisation.